ui/cocoa/main.m

```diff
@@ -680,6 +680,13 @@ static void cocoa_display_init(DisplayState *ds, DisplayOptions *opts)
     COCOA_DEBUG("qemu_cocoa: cocoa_display_init\n");
 
     screen.cursor_show = opts->has_show_cursor && opts->show_cursor;
+    screen.full_grab = opts->u.cocoa.full_grab;
+
+    if (opts->u.cocoa.has_swap_option_command) {
+        screen.swap_option_command = opts->u.cocoa.swap_option_command;
+    } else {
+        screen.swap_option_command = true;
+    }
```

Comment:
`screen.swap_option_command` always becoms true.

Reply:
It should not, and I'm pretty sure I tested that both worked. If it has_swap_option_command, then it uses the value that was provided. Otherwise it sets a default.

Reply:
I read `has_swap_option_command` as `swap_option_command`. My bad.